docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -1738,10 +1722,8 @@ hasSubsequenceCaseInsensitiveUTF8(haystack, needle)
 
 **Returned values**
 
-- 1, if needle is a subsequence of haystack.
-- 0, otherwise.
-
-Type: `UInt8`.
+- 1, if needle is a subsequence of haystack. [UInt8](../data-types/int-uint.md).
```

Comment:
Let's change this into a single line to avoid the impression that two values (in the sense of an array or a tuple) are returned:

`- 1, if needle is a subsequence of haystack, 0 otherwise [UInt8](../data-types/int-uint.md).`

The same applies to other places in this PR, e.g. l. 1689.

Reply:
Another question - should numbers in the return type be in \` \` quotations? eg. - 

`1`, if needle is a subsequence of haystack, `0` otherwise...